man/sd_bus_error.xml

```diff
@@ -138,234 +150,188 @@
       </funcprototype>
     </funcsynopsis>
 
-    <para>
-      <constant>SD_BUS_ERROR_FAILED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NO_MEMORY</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_SERVICE_UNKNOWN</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NAME_HAS_NO_OWNER</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NO_REPLY</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_IO_ERROR</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_BAD_ADDRESS</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NOT_SUPPORTED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_LIMITS_EXCEEDED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_ACCESS_DENIED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_AUTH_FAILED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NO_SERVER</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_TIMEOUT</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_NO_NETWORK</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_ADDRESS_IN_USE</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_DISCONNECTED</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_INVALID_ARGS</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_FILE_NOT_FOUND</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_FILE_EXISTS</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_UNKNOWN_METHOD</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_UNKNOWN_OBJECT</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_UNKNOWN_INTERFACE</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_UNKNOWN_PROPERTY</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_PROPERTY_READ_ONLY</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_UNIX_PROCESS_ID_UNKNOWN</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_INVALID_SIGNATURE</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_INCONSISTENT_MESSAGE</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_MATCH_RULE_NOT_FOUND</constant>
-    </para>
-    <para>
-      <constant>SD_BUS_ERROR_MATCH_RULE_INVALID</constant>
-    </para>
-
   </refsynopsisdiv>
 
   <refsect1>
     <title>Description</title>
 
     <para>The <structname>sd_bus_error</structname> structure carries
-    information for a <filename>sd-bus</filename> error. The
-    functions described below can be used to set and query fields in
-    this structure. The <structfield>name</structfield> field contains a
-    short identifier of an error. It should follow the rules for error
-    names described in the D-Bus specification, subsection <ulink
+    information about a D-Bus error condition. The functions described
```

Comment:
Hmm, what's the rationale for removing the "the"? Sounds weird that way. I mean, it's really all the functions the man page describes that are included in the library?